### PR TITLE
feat: add Github Actions to release clusterloader2 

### PR DESCRIPTION
**Pull request by alyssa1303**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

This PR is to add Github Actions to release clusterloader2

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: .github/clusterloader2-release.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  release:
+    types: [published, edited]
+
+env:
+  IMAGE_REPO: ghcr.io/${{ github.repository_owner }}
```
Comment (ritwikranjan) [inline .github/clusterloader2-release.yml:12]: I actually got into an issue for the same, `github.repository_owner` is `Azure` here with A as uppercase, however docker repo cannot contain uppercase. You can check #5 to see the fix.

```suggestion
  IMAGE_REPO: ghcr.io/${{ github.repository_owner.toLowerCase }}
```

File: .github/clusterloader2-release.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name: Release ClusterLoader2
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  release:
+    types: [published, edited]
```
Comment (ritwikranjan) [inline .github/clusterloader2-release.yml:9]: I guess only one of them would do no?

**Review by ritwikranjan — CHANGES_REQUESTED** [review]

File: .github/clusterloader2-release.yml (hunk 1 above)
Comment (alyssa1303) [reply .github/clusterloader2-release.yml]: fixed now

File: .github/clusterloader2-release.yml (hunk 2 above)
Comment (alyssa1303) [reply .github/clusterloader2-release.yml]: removed this since I'm going with tag and path

**Review by ritwikranjan — APPROVED** [review]

**Review by sumanthreddy29 — APPROVED** [review]

**Pull request merged**